task2/src/main/java/ElectricalAppliances/CleaningDevice/Dry/HairDryer/HairDryer.java

```diff
@@ -0,0 +1,36 @@
+package ElectricalAppliances.CleaningDevice.Dry.HairDryer;
+
+import ElectricalAppliances.CleaningDevice.Dry.AirForceAdjustment;
+import ElectricalAppliances.CleaningDevice.Dry.Dry;
+
+
+public class HairDryer extends Dry implements AirForceAdjustment {
+    protected int hairDryerPower;
+    protected String caseColor;
+    protected boolean coldAir;
+
+    public HairDryer(int hairDryerPower, String caseColor, boolean coldAir, int speed, boolean power, int voltagePower, double weight, double length, double width, double thickness, int releaseDate) {
+        super(speed, power, voltagePower, weight, length, width, thickness, releaseDate);
+
+        this.hairDryerPower = hairDryerPower;
+        this.caseColor = caseColor;
+
+        bulkyItemEvaluation(weight, length, width, thickness);
```

Comment:
Unnecessary code because 'bulkyItem' will be set automatically in parent constructor